### ISSUE TITLE
Support for Array

### ISSUE THREAD

**Issue by beeing** [davidcalhoun/jstoxml#33]
Hi, I like your library which is simple and straight-forward.

However, I was caught by surprised when the array being converted to xml.

```js
const { toXML } = require('jstoxml')
console.log(toXML({ x: ['lat','lng'] })) // -> <x>latlng</x>
console.log(toXML({ x: [1,2,3] })) // -> <x>123</x>
```

The library will just concat all the values into a string instead of separating it, presumably like:

```xml
<x-arr name="x">
    <x>1</x>
    <x>2</x>
    <x>3</x>
</x-arr>
```

Is there anything that I missed?

Thanks in advance.

**Comment by davidcalhoun** [comment]
Thanks for using the library!  That's an interesting suggestion, I can definitely see how that would be handy.  I'll leave this open and consider adding in this feature.

Currently you can achieve what you need with this (see the duplicate keys example in the readme as well):

```js
toXML([
  { x: 1 },
  { x: 2 },
  { x: 3 }
]);

// -> '<x>1</x><x>2</x><x>3</x>'
```

**Comment by jimmdd** [comment]
I like your library but I need this feature! I have to convert JavaScript object with nested array like data"{name:sth, risk:[{sth},{sth}]} and convert to <data><name>sth</name><risk>sth</risk><risk>sth</risk></data>. With current version, I can't do this conversion without rewrite the entire json to use _name and _content.

**Comment by davidcalhoun** [comment]
Thanks for the feedback.  This is now implemented in `1.4.0`
